src/dbnode/storage/index.go

```diff
@@ -320,6 +323,8 @@ func newNamespaceIndexWithOptions(
 
 		queryWorkersPool: newIndexOpts.opts.QueryIDsWorkerPool(),
 		metrics:          newNamespaceIndexMetrics(indexOpts, instrumentOpts),
+
+		debugSkipIndexEvery: newIndexOpts.opts.DebugSkipIndexEvery(),
```

Comment:
Maybe we should leave a TODO here to remove if we can figure out a better way to write the tests.

Reply:
Actually went and made a more useful thing which is to not index documents with certain field name/value pairs and using that now.